lib/managers/external_system_status_manager.dart

```diff
@@ -0,0 +1,113 @@
+import 'dart:async';
+import 'dart:io';
+import 'package:http/http.dart' as http;
+import 'package:mobx/mobx.dart';
+import 'package:momento_booth/main.dart';
+import 'package:momento_booth/managers/settings_manager.dart';
+import 'package:momento_booth/models/external_system_status.dart';
+import 'package:momento_booth/models/settings.dart';
+import 'package:momento_booth/models/subsystem_status.dart';
+import 'package:momento_booth/utils/logger.dart';
+
+export 'package:momento_booth/models/external_system_status.dart';
+
+part 'external_system_status_manager.g.dart';
+
+class ExternalSystemStatusManager = ExternalSystemStatusManagerBase with _$ExternalSystemStatusManager;
+
+abstract class ExternalSystemStatusManagerBase with Store, Logger {
+  @observable
+  ObservableList<ExternalSystemStatus> systems = ObservableList<ExternalSystemStatus>();
+  Timer? _checkTimer;
+
+  ObservableList<SubsystemStatus> get systemStatuses => ObservableList<SubsystemStatus>.of(systems.map((status) => status.isHealthy));
+
+  ExternalSystemStatusManagerBase();
+
+  /// Initializes the manager, setting up the timer for periodic checks.
+  void initialize() {
```

Comment:
You might be able to use the `autorun` mechanism (see `MqttManager` for an example) to automatically rerun the initialization when the settings have changed.